clients/fides-js/src/fides.ts

```diff
@@ -249,6 +255,49 @@ async function init(this: FidesGlobal, providedConfig?: FidesConfig) {
   });
   Object.assign(this, updatedFides);
   updateWindowFides(this);
+
+  // Now that we have the final experience, we can save OneTrust preferences to the API, if they exist
+  if (
```

Comment:
I think it might be work breaking all of the OT migration logic out of this file and into a dedicated file of its own. Similar to my previous comment, how much of this is specific to OT and how much of it can we make generic for future competitor support?

Reply:
I think that's a great suggestion! I'll at least break out into a separate file for now